src/app/modules/create-association/create-association.component.scss

```diff
@@ -0,0 +1,188 @@
+
+$light: 300;
+$regular: 400;
+$medium: 500;
+$semiBold: 600;
+$bold: 700;
+
+$white: #eeeeee;
+$yellow: #ffd369;
+$blue-dark: #222831;
+
+.container-create-association {
+  height: 100vh;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.container-create-association .container-layout {
+  height: 100%;
+  width: calc(100% - 340px);
+}
+
+.container-create-association .top {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 150px;
+}
+
+.container-create-association .top .title {
+  color: $white;
+  font-family: "Quicksand", sans-serif;
+  font-weight: $bold;
+  font-size: 20px;
+}
+
+.container-create-association .top .get-app-btn {
+  border: 0;
+  background: $blue-dark;
+  padding: 10px 20px;
+  border-radius: 20px;
+  color: $white;
+  cursor: pointer;
+}
+
+.container-create-association .bottom {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px 150px;
+  transform: translate(0%, 50%);
+}
+
+form {
+  flex-direction: column;
+  align-items: center;
+  height: 400px;
+  display: flex;
+  justify-content: center;
+}
+
+form .container-association {
+  width: 100%;
+  justify-content: space-between;
+  margin: 20px 0;
+  display: flex;
+  flex-direction: column;
+}
+
+form .container-association .container-input input {
+  border: 0;
+  background: transparent;
+  border-bottom: 1px solid #eeeeee;
+  color: $white;
+}
+
+form .container-association .container-input input.ng-invalid.ng-touched {
+  border-bottom: 1px solid red;
+}
+
+form .container-association span {
+  color: $white;
+  font-family: "Quicksand", sans-serif;
+  font-weight: $regular;
+  font-size: 16px;
+  margin-top: 15px;
+}
+
+form label {
+  margin-right: 10px;
+  color: $white;
+  font-family: "Quicksand", sans-serif;
+  font-weight: $medium;
+  font-size: 16px;
+}
+
+form .container-apartments {
+  width: 100%;
+  margin: 20px 0;
+  display: flex;
+  flex-direction: column;
+}
+
+form .container-input-apartments {
+  overflow-y: auto;
+  width: 100%;
+  max-height: 100px;
+}
+
+form .container-apartments .container-input {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+form .container-apartments .container-input input {
+  border: 0;
+  background: transparent;
+  border-bottom: 1px solid #eeeeee;
+  color: $white;
+}
+
+form .container-apartments .container-input input.ng-invalid.ng-touched {
+  border-bottom: 1px solid red;
+}
+
+form .container-apartments .apartment-message {
+  color: $white;
+  font-family: "Quicksand", sans-serif;
+  font-weight: $regular;
+  font-size: 16px;
+  margin-top: 15px;
+}
+
+form span {
+  color: $white;
+  font-family: "Quicksand", sans-serif;
+  font-weight: $regular;
+  font-size: 16px;
+}
+
+form span .add-apartment-btn {
+  border: 0;
+  padding: 5px 7px;
+  border-radius: 3px;
+  background-color: $blue-dark;
+  color: $white;
+  cursor: pointer;
+  margin: 20px 0;
+  margin-right: 10px;
+}
+
+form .submit-btn {
```

Comment:
din moment ce folosesti scss, de ce nu le pui toate in interiorul form, ca sa nu repeti form de n ori?

form {
  bla bla

  .submit-btn {
   bla bla
  }
}